datagrid/deploy.sh

```diff
@@ -23,7 +33,6 @@ while [ "$(oc get statefulset datagrid-service -o jsonpath='{.status.readyReplic
     echo "Waiting for statefulset to have $INSTANCES readyReplicas"
     sleep 5
 done
-oc exec datagrid-service-0 -- curl -X POST -d @/opt/datagrid/standalone/configuration/game-config.json -H application/json http://datagrid-service.datagrid-demo.svc.cluster.local:8080/rest/game/game
 
-# Set a native memory baseline
-./diagnostics/native-baseline.sh
+GAME_CONFIG_JSON=$(cat "$RESOURCE_DIR/config/game-config.json" | tr -d "\n")
+oc exec datagrid-service-0 -- curl -X POST -d "$GAME_CONFIG_JSON" -H application/json http://datagrid-service-0:8080/rest/game/game
```

Comment:
When pairing down datagrid to a single pod for small demos, the pod could not curl to the service (`datagrid-service.datagrid-demo.svc.cluster.local:8080`) for some reason, and it was timing out.  For that reason I just curled to the pod:port (`datagrid-service-0:8080`).  If there's a better way to do this, please let me know.